src/layer/tile/GridLayer.js

```diff
@@ -228,7 +228,7 @@ export const GridLayer = Layer.extend({
 	redraw() {
 		if (this._map) {
 			this._removeAllTiles();
-			const tileZoom = this._clampZoom(this._map.getZoom());
+			const tileZoom = Math.round(this._clampZoom(this._map.getZoom()));
```

Comment:
There's gonna be an edge case when the `GridLayer`'s `minZoom` or `maxZoom` are fractional as well - the `tileZoom` might fall outside of the clamp.

I don't think this will be a problem since I don't know any case where people use fractional `min`/`maxZoom`. Perhaps a note in the docstrings for those options would be enough?

Reply:
Hmm, not familiar with the use cases, but that's a good catch. If there is a case where `minNativeZoom` is `2.2` and `maxNativeZoom` is `2.9`, then the `tileZoom` will always be out of bound. I will wait for the team to decide on this one.

Reply:
As the examples are from tileLayer, and there could be plugins where people redraw differently at fractional zoom levels, can we just change L.TileLayer and leave gridLayer as is? (Then there is no issue if plugins want to use fractional zoom levels)